MAINTAINERS

```diff
@@ -0,0 +1,22 @@
+Adam Badura <abadura@o2.pl>
+BCRodich <RodichBruce@gmail.com>
+Bill Rosgen <rosgen@gmail.com>
+Brett Lentz <blentz@users.noreply.github.com>
+Chris Noon <chrisnoo@msn.com>
+Dr. Martin Brumm <neutronc@users.noreply.github.com>
```

Comment:
please change to dr.martin.brumm@googlemail.com

Reply:
```suggestion
Dr. Martin Brumm <dr.martin.brumm@googlemail.com>
```